### PR TITLE
[WIP] feat(design): Add blue-gray palette to theming

### LINKED ISSUE #151: (feat): specify a default color hue in a palette
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[x] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Environment

<pre><code>
Daffodil version: X.Y.Z

 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

### PR DESCRIPTION
**Pull request by xelaint**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: Fixes #151 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

File: libs/design/src/scss/theming/_theming.scss [hunk 1]
```diff
@@ -1,7 +1,8 @@
 @import 'variables';
 @import 'color';
 
-$primary: daff-color($daff-primary, 500);
-$accent: daff-color($daff-accent, 500);
-$gray: daff-color($daff-gray, 500);
-$black: daff-color($daff-gray, 900);
\ No newline at end of file
+$primary: daff-color($daff-yellow, 60);
+$accent-purple: daff-color($daff-purple, 60);
```
Comment (damienwebdev) [inline libs/design/src/scss/theming/_theming.scss]: I disagree with the naming here.

```
primary
accent
???????
```

File: libs/design/src/atoms/button/button.component.scss [hunk 2]
```diff
@@ -38,9 +38,9 @@
         }
     }
 
-    &.daff-accent-blue {
-        background-color: $accent-blue;
-        border: 1px solid $accent-blue;
+    &.daff-blue {
```
Comment (damienwebdev) [inline libs/design/src/atoms/button/button.component.scss]: Still no.

File: libs/design/src/core/colorable/colorable.spec.ts [hunk 3]
```diff
@@ -111,6 +111,7 @@ describe('colorInPalette', () => {
         expect(colorInPalette("white")).toEqual(true);
         expect(colorInPalette("primary")).toEqual(true);
         expect(colorInPalette("accent")).toEqual(true);
+        expect(colorInPalette("blue")).toEqual(true);
```
Comment (damienwebdev) [inline libs/design/src/core/colorable/colorable.spec.ts]: I don't think we should allow arbitrary colors here. We allow white/black as those are special use-cases, but I think we should stick to a few theme colors.

File: libs/design/src/atoms/button/button.component.scss (hunk 2 above)
Comment (xelaint) [reply libs/design/src/atoms/button/button.component.scss]: We didn't talk about this part yesterday. How do we style colors that are not $primary and $accent then? I still want to be able to add a `blue` variant to button, hero, callout.

File: libs/design/src/core/colorable/colorable.spec.ts (hunk 3 above)
Comment (xelaint) [reply libs/design/src/core/colorable/colorable.spec.ts]: Same comment

> We didn't talk about this part yesterday. How do we style colors that are not $primary and $accent then? I still want to be able to add a `blue` variant to button, hero, callout.

**Comment by xelaint** [comment]
Need to address issue #151

File: libs/design/src/atoms/button/button.component.scss (hunk 2 above)
Comment (damienwebdev) [reply libs/design/src/atoms/button/button.component.scss]: If you want, you can add a "tertiary" color to the palette.

**Comment by xelaint** [comment]
Issue moved to PR #185

**Pull request closed without merging**